mobs/owl.lua

```diff
@@ -1,3 +1,46 @@
+local follow = {}
```

Comment:
I presume the commit that brought this in is same as PR #8 , same coment applies

Reply:
i will remove this then.. i just merged from the #8